ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.service.ts

```diff
@@ -132,7 +132,7 @@ export class AddressInputService implements OnDestroy {
       // todo maybe some way to introduce derived updates and make it so state is updated atomically?
       this.addressForm.valueChanges.pipe(
         distinctUntilChanged((x, y) => _.isEqual(x, y)),
-        concatMap((formValue) => {
+        switchMap((formValue) => {
```

Comment:
The  change of "country" from form (that was avoiding switching the state/province field) was stuck right here.
Best explanation I have is an rxjs bug in concatMap: the concatMap function was not executing.
I first tried updating the library and it appeared to fix the problem.
switchMap with either version was working to and is also an acceptable option here.